requirements.txt

```diff
@@ -0,0 +1,21 @@
+tqdm==4.64.1
```

Comment:
Most of these requirements are in the `stable-diffusion-sdkit` package. https://github.com/easydiffusion/sdkit/blob/main/setup.py#L5

And https://github.com/easydiffusion/stable-diffusion/blob/main/requirements.txt

Is it necessary to define those here again?

Reply:
that was intended to be there for local development. If you rather not have them defined in a separate file, I need to find an alternative. Either use the setup file somehow or clearly document that it is for development only.

The stable-diffusion repo is completely distinct from this one isn't it?

Reply:
Sorry, not sure I understand. `setup.py` is how this project is configured, it's not for development-only. That's how pip installs the dependencies today on users' PCs.

Yes, https://github.com/easydiffusion/stable-diffusion is different from this repo, but we depend on it. That repo is just a thin wrapper over the official stable-diffusion repo, in order to allow installing stable-diffusion's code via pip.

Doesn't setup.py work for you?

Reply:
Ah sorry, that's my lack of experience dealing with `setup` files that made me doing that. 

reverted in a2dbf02f7432ca1bd5b283fd58960a6ad8c41199

Reply:
Thanks, no worries!